src/app/user/RestaurantCard.vue

```
@@ -0,0 +1,190 @@
+
+    let detachStripe: Unsubscribe | null = null;
+
+    // Fetch shop info to get owner UID
+    const fetchShopInfo = async () => {
+      try {
+        const shopDoc = await getDoc(doc(db, `restaurants/${restaurantId.value}`));
+        if (shopDoc.exists()) {
+          shopInfo.value = shopDoc.data();
+          ownerUid.value = shopInfo.value.uid;
+          // Start monitoring card info once we have owner UID
+          checkStripe();
```

ESLint

[error] 'checkStripe' was used before it was defined.

(no-use-before-define)